apps/frontend/src/app/pages/company/create-profile/create-profile.component.html

```diff
@@ -0,0 +1,366 @@
+              (click)="toggleEditProfileSectionOne()"
+            >
+              Edit <i class="bi bi-pencil text-primary pl-2"></i>
+            </button>
+          </div>
+          }@else{
+          <div class="pt-6 pb-[50px] flex justify-end">
+            <button
+              class="text-base-100 font-bold font-mono"
+              (click)="toggleEditProfileSectionOne()"
+            >
+              Update <i class="bi bi-check2 text-primary pl-2"></i>
+            </button>
+          </div>
+          }
+          <div>
+            <p class="text-primary font-bold font-mono pb-4">
+              What your are looking for?
+            </p>
+            @if(editProfileSectionTwo){
+            <div>
+              <wagademy-input
+                formControlName="newWord"
+                (keyup.enter)="addAreaOfInterest()"
+              ></wagademy-input>
+              <div class="flex flex-wrap gap-4 pt-6">
+                @for(word of areaOfInterest; track addAreaOfInterest; let i =
+                $index){
+                <div
+                  class="bg-neutral flex gap-2 rounded-3xl font-sm font-mono py-1 px-2"
+                >
+                  <p class="text-base-100">{{ word }}</p>
+                  <i
+                    (click)="removeAreaOfInterest(i)"
+                    class="bi bi-x-lg text-error cursor-pointer"
+                  ></i>
+                </div>
+                }
+              </div>
+            </div>
+            } @if(!editProfileSectionTwo){
+            <div class="flex flex-wrap gap-4">
+              @for(word of areaOfInterest; track areaOfInterest){
+              <div
+                class="bg-neutral rounded-3xl text-base-100 font-sm font-mono py-1 px-2"
+              >
+                {{ word }}
+              </div>
+              }
+            </div>
+            } @if (!editProfileSectionTwo) {
+            <div class="pt-6 pb-[50px] flex justify-end">
+              <button
+                class="text-base-100 font-bold font-mono"
+                (click)="toggleEditProfileSectionTwo()"
+              >
+                Edit <i class="bi bi-pencil text-primary pl-2"></i>
+              </button>
+            </div>
+            }@else{
+            <div class="pt-6 pb-[50px] flex justify-end">
+              <button
+                class="text-base-100 font-bold font-mono"
+                (click)="toggleEditProfileSectionTwo()"
+              >
+                Update <i class="bi bi-check2 text-primary pl-2"></i>
+              </button>
+            </div>
+            }
+            <div class="flex justify-center">
+              <button
+                class="btn btn-primary w-[200px]"
+                (click)="saveAndConfirm()"
+              >
+                Save And Confirm
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  }
+</form>
+<dialog id="create_job_modal" class="modal">
```

Comment:
Also, you can remove this job modal, this is already on hiring page, i think you added this due to a misunderstanding, remove it to from the typescript component